src/ansiblelint/schemas/changelog.json

```diff
@@ -57,6 +123,100 @@
           },
           "type": "array"
         },
+        "objects": {
+          "additionalProperties": false,
+          "properties": {
+            "playbook": {
+              "items": {
+                "properties": {
+                  "description": {
+                    "markdownDescription": "Value of `short_description` from plugin `DOCUMENTATION`.",
+                    "title": "Description",
+                    "type": "string"
+                  },
+                  "name": {
+                    "markdownDescription": "It must not be the FQCN, but the name inside the collection.",
+                    "pattern": "[a-zA-Z0-9_]+",
+                    "title": "Short playbook name",
+                    "type": "string"
+                  },
+                  "namespace": {
+                    "type": "null"
+                  }
+                },
+                "type": "object"
+              },
+              "type": "array"
+            },
+            "role": {
+              "items": {
+                "properties": {
+                  "description": {
+                    "markdownDescription": "Value of `short_description` from plugin `DOCUMENTATION`.",
+                    "title": "Description",
+                    "type": "string"
+                  },
+                  "name": {
+                    "markdownDescription": "It must not be the FQCN, but the name inside the collection.",
+                    "pattern": "[a-zA-Z0-9_]+",
+                    "title": "Short role name",
+                    "type": "string"
+                  },
+                  "namespace": {
+                    "type": "null"
+                  }
+                },
+                "type": "object"
+              },
+              "type": "array"
+            }
+          },
+          "type": "object"
+        },
+        "plugins": {
+          "additionalProperties": false,
+          "properties": {
+            "become": {
+              "$ref": "#/$defs/plugin-descriptions"
+            },
+            "cache": {
+              "$ref": "#/$defs/plugin-descriptions"
+            },
+            "callback": {
+              "$ref": "#/$defs/plugin-descriptions"
+            },
+            "cliconf": {
+              "$ref": "#/$defs/plugin-descriptions"
+            },
+            "connections": {
+              "$ref": "#/$defs/plugin-descriptions"
+            },
+            "filter": {},
```

Comment:
Why is this empty?

Reply:
The doc did not give any details regarding what filter allows.

Reply:
Exactly the same as every other plugin type.